src/async-components/views/dialogs/security/ExportE2eKeysDialog.tsx

```diff
@@ -46,6 +48,9 @@ interface IState {
 type AnyPassphrase = KeysStartingWith<IState, "passphrase">;
 
 export default class ExportE2eKeysDialog extends React.Component<IProps, IState> {
```

Comment:
https://github.com/vector-im/element-web/issues/9478 says:

> Also, the wording needs to be clearere that this is a request for a *new* passphrase, rather than expecting an existing account password.

Does that predate the current wording in the dialog? Or is rewording still needed?

Reply:
Good spot